File: Php/poo/figure/library/triangle.php
<?php

        class triangle extends figureGeometric{
        
        private $base;
        private $height;
        private $calculate;
            
    public function __construct($base,$height){
        parent::__construct("Triangle");
        $this->base = $base;
        $this->height = $height;
    }

    public function getBase(){
        return $this->base;
    }

    public function getHeight(){
        return $this->height;
    }

    public function calculateArea(): float {
        $this->calculate = ($this->getBase() * $this->getHeight()) / 2;
        return $this->calculate;
    }
}



    
    
    

File: Java/poo/figure/library/triangle.java
package library;

public class triangle extends figureGeometric {
    private double base;
    private double height;

    public triangle(double base, double height) {
        super("Rectángulo");
        this.base = base;
        this.height = height;
    }

    public double getBase() {
        return this.base;
    }

    public double getHeight() {
        return this.height;
    }

    @Override
    public double calculateArea() {
        return (this.getBase() * this.getHeight())/2;
    }
}